qa/tasks/ceph.py

```diff
@@ -124,6 +124,60 @@ def ceph_crash(ctx, config):
                     pass
 
 
+class Rotater(object):
```

Comment:
Rotator?

(Maybe: Ro'tater -- a linear sequence of Andean root vegetables?
  ... sorry I'll see myself out)